vite.config.ts

```diff
@@ -43,7 +43,7 @@ export default defineConfig({
         sourcemap: true,
         minify: true,
         rollupOptions: {
-            input: [...componentsPath.map((element) => element.path), "src/styles.css"],
+            input: [...componentsPath.map((element) => element.path), "src/styles.css", "tailwind.config.js"],
```

Comment:
Probably not needed as it is already a `.js` file and use `require`, wdyt?

Reply:
I added it to ensure `tailwind.config.js` is copied to `/dist` is there another way to achieve it?

Reply:
Make sense 👍

Reply:
Thanks for double-checking 🙌